docs/application/tizen-studio/native-tools/manifest-text-editor.md

```diff
@@ -32,50 +32,65 @@ There are 2 different ways to edit the `tizen-manifest.xml` file:
+|              |                                  |                         | `<icon>`         |
+|              |                                  |                         | `<label>`        |
+|              |                                  |                         | `<support-size>` |
+|              | `<component-based-application>`  |                         |                  |
+|              |                                  | `<label>`               |                  |
+|              |                                  | `<icon>`                |                  |
+|              |                                  | `<app-control>`         |                  |
+|              |                                  | `<metadata>`            |                  |
+|              |                                  | `<datacontrol>`         |                  |
+|              |                                  | `<background-category>` |                  |
+|              |                                  | `<splash-screens>`      |                  |
+|              |                                  |                         | `<splash-screen>`|
+|              |                                  | `<frame-component>`     |                  |
+|              |                                  |                         | `<label>`        |
+|              |                                  |                         | `<icon>`         |
+|              |                                  | `<service-component>`   |                  |
+|              |                                  |                         | `<label>`        |
+|              |                                  |                         | `<icon>`         |
+|              | `<shortcut-list>`                |                         |                  |
+|              |                                  | `<shortcut>`            |                  |
+|              |                                  |                         | `<icon>`         |
+|              |                                  |                         | `<label>`        |
+|              | `<account>`                      |                         |                  |
+|              |                                  | `<account-provider>`    |                  |
+|              |                                  |                         | `<icon>`         |
+|              |                                  |                         | `<label>`        |
+|              |                                  |                         | `<capability>`   |
+|              | `<privileges>`                   |                         |                  |
+|              |                                  | `<privilege>`           |                  |
+|              | `<feature>`                      |                         |                  |
```

Comment:
I've been fixed code indentation.
It doesn't effect about the link.
As I checked, all links don't work.
Please check it.

Reply:
When we set up this repository, during the converting from html file to md file, we removed the links on purpose because of the readability and managing issue.
It is correct that now no links are there.